package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "usng.js",
+  "version": "0.0.1",
+  "devDependencies": {
+    "chai": "^1.10.0",
+    "grunt": "latest",
```

Comment:
Is this safe?  What if it updated to a version of grunt that is not backwards compatible with the other dev dependencies?